api/integreatly/v1alpha1/grafana_types.go

```diff
@@ -504,6 +504,7 @@ type GrafanaConfigDashboards struct {
 	// +nullable
 	VersionsToKeep           *int   `json:"versions_to_keep,omitempty" ini:"versions_to_keep,omitempty"`
 	DefaultHomeDashboardPath string `json:"default_home_dashboard_path,omitempty" ini:"default_home_dashboard_path,omitempty"`
+	MinRefreshInterval       string `json:"min_refresh_interval,omitempty" ini:"min_refresh_interval,omitempty"`
```

Comment:
We haven't been the greatest at this but trying to become better. Can you also write a comment above what this config values does?

Reply:
Sure thanks for Review will do that